test-integration/helpers/test-element.coffee

```diff
@@ -51,7 +51,8 @@ Object.defineProperties TestItemHelper.prototype,
 
 
 class TestHelper extends TestItemHelper
-  constructor: (test, testElementLocator, commonElements = {}, options) ->
+  constructor: (test, testElementLocator, commonElements, options) ->
+    commonElements ||= _.result(@, 'elementRefs', {})
     defaultOptions =
```

Comment:
okay, i don't see why not have both options.  i'd kept it out of the class because i felt it would keep it cleaner, but attaching it makes more sense for reference the locators throughout the helper later on if needed